stdlib/2.7/argparse.pyi

```diff
@@ -35,25 +35,28 @@ class RawTextHelpFormatter(RawDescriptionHelpFormatter): ...
 class ArgumentDefaultsHelpFormatter(HelpFormatter): ...
 
 class ArgumentError(Exception):
-    argument_name = ... # type: Any
-    message = ... # type: Any
+    argument_name = ...  # type: Any
+    message = ...  # type: Any
     def __init__(self, argument, message) -> None: ...
 
 class ArgumentTypeError(Exception): ...
 
 class Action(_AttributeHolder):
-    option_strings = ... # type: Any
-    dest = ... # type: Any
-    nargs = ... # type: Any
-    const = ... # type: Any
-    default = ... # type: Any
-    type = ... # type: Any
-    choices = ... # type: Any
-    required = ... # type: Any
-    help = ... # type: Any
-    metavar = ... # type: Any
-    def __init__(self, option_strings, dest, nargs=..., const=..., default=..., type=...,
-                 choices=..., required=..., help=..., metavar=...): ...
+    option_strings = ...  # type: Any
+    dest = ...  # type: Any
+    nargs = ...  # type: Any
+    const = ...  # type: Any
+    default = ...  # type: Any
+    type = ...  # type: Any
+    choices = ...  # type: Any
+    required = ...  # type: Any
+    help = ...  # type: Any
+    metavar = ...  # type: Any
+    def __init__(self, option_strings: List[str], dest=str,
+                 nargs: Union[int, str]=..., const: Any =...,
+                 default: Any =..., type: Callable[[str], Any] =...,
+                 choices: Iterable[Any] =..., required: bool=...,
+                 help: str=..., metavar: str =...) -> None: ...
```

Comment:
Also PEP 8 requires spaces on both side of the `=` when a type annotation and a default value are combined.